### PR TITLE
blog: update 10.0.0 notable changes

### PR DESCRIPTION
**Pull request by BridgeAR**
Refs: https://github.com/nodejs/node/pull/20316

@jasnell PTAL

### CODE REVIEW

File: locale/en/blog/release/v10.0.0.md [hunk 1]
```diff
@@ -30,18 +30,20 @@ The full set of changes for the Node.js 10.0.0 release are detailed below.
 
 ### Notable Changes
 
-* The N-API native addons API has graduated from experimental status.
```
Comment (mhdawson) [inline locale/en/blog/release/v10.0.0.md:33]: Just wondering about the removal of this one?

Comment (BridgeAR) [reply locale/en/blog/release/v10.0.0.md:33]: It is in the list below and I was not sure why it was duplicated here. It is the only entry that is not aligned with the changelog.

Comment (mhdawson) [reply locale/en/blog/release/v10.0.0.md:33]: @BridgeAR I must be missing something.  When I hit "view" and search on graduated I don't see it anywhere after the removal?

Comment (BridgeAR) [reply locale/en/blog/release/v10.0.0.md:33]: https://github.com/BridgeAR/nodejs.org/blame/4be9f4c6e7dc11e068cba19c86e8dd797e5f4a11/locale/en/blog/release/v10.0.0.md#L71

Comment (mhdawson) [reply locale/en/blog/release/v10.0.0.md:33]: @BridgeAR thanks, see that is has different wording.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by daxlab — APPROVED** [review]

**Review by fhinkel — APPROVED** [review]

**Review by ckotzbauer — APPROVED** [review]

**Review by hiroppy — APPROVED** [review]

**Pull request merged**